libs/auth/src/signed_hash_validation.ts

```diff
@@ -80,12 +82,16 @@ export async function generateSignedHashValidationQrCodeValue(
         electionRecord.electionPackageHash
       )
     : '';
+  const pollbookPackageHash = machineState.pollbookPackageHash
+    ? formatPollbookPackageHash(machineState.pollbookPackageHash)
+    : '';
   const date = new Date();
 
   const messagePayloadParts: string[] = [
     systemHash,
     softwareVersion,
     combinedElectionHash,
+    pollbookPackageHash,
     date.toISOString(),
   ];
```

Comment:
In line with my comment below, rather than returning both a `combinedElectionHash` and a `pollbookPackageHash`, we could return a single `configurationId` that's either an election hash or a poll book package hash. That might abstract things better and will also avoid us having to edit and make backwards compatible the QR code parsing logic in the Signed Hash Validation web app run at https://check.voting.works: https://github.com/votingworks/livecheck-web/blob/42c9a78cf062cbc10c478a1a91ac964825676c84/server/livecheck.py#L115-L142